search-app/docs/api.md

```diff
@@ -5405,3 +5412,152 @@ Example un-healthy response body:
   }
 }
 ```
+#### <a name="generic-association"></a> Associate concepts
+
+A concept of any type can be associated to another concept of any type using the association API. Note this type of association is functionally the same as associations of tools, services, and variables to collections but, uses a different API route and is applicable to all possible concept relationships. In such fashion, any concept can be associated to any other concept, including concepts of the same type i.e. associate a variable to another variable. Associations can also include an optional data payload the purpose of which is to describe the association itself. Associations which do not initially have association details, may have details added simply by providing association details as a subsequent revision. A concept cannot be associated to itself.
+
+__Example__
+
+```
+curl -v -XPOST -H "Authorization: Bearer XXXXX" -H "Content-Type:application/json" "%CMR-ENDPOINT%/associate/GRD1200000000-PROV1" -d '[{"concept_id": "S1200000001-PROV1"}]'
+```
+__Example Response__
+```
+[{
+  "generic_association" : {
+    "concept_id" : "GA1200000000-CMR",
+    "revision_id" : 3
+  },
+  "associated_item" : {
+    "concept_id" : "S1200000001-PROV1"
+  }}]
+```
+
```

Comment:
give a "payload" example. and don't we call it just "data" in the json? Maybe we should not call it "payload" in the description as that is more of an internal title.